### PR TITLE
implement apis list a user's project membership and members of a project

### PR DESCRIPTION
**Pull request by dvoet**
- implement missing api to list all members of a project and their roles (owner only)
- change /user/billing api to include role for each project as per @MattPutnam 
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Make sure liquibase is updated if appropriate
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] **Submitter**: If PR includes new or changed db queries, include the explain plans in the description
- [x] **Submitter**: Update FISMA documentation if changes to:
  - Authentication
  - Authorization
  - Encryption
  - Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell  that the PR exists if he wants to look at it **(apply requires_doge label)**
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- Review cycle:
  - LR reviews
  - Rest of team may comment on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH
  - Submitter updates documentation as needed
  - Submitter **reassigns to LR** for further feedback
- [x]  sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by dvoet** [comment]
explain for new query

```
MySQL [rawls]> explain select x2.`EMAIL`, x3.`ROLE` from `PROJECT_USERS` x3, `USER` x2 where (x3.`PROJECT_NAME` = 'new_project') and (x2.`SUBJECT_ID` = x3.`USER_SUBJECT_ID`);
+----+-------------+-------+--------+----------------------------------+--------------------------+---------+--------------------------+------+-----------------------+
| id | select_type | table | type   | possible_keys                    | key                      | key_len | ref                      | rows | Extra                 |
+----+-------------+-------+--------+----------------------------------+--------------------------+---------+--------------------------+------+-----------------------+
|  1 | SIMPLE      | x3    | ref    | PRIMARY,FK_PROJECT_USERS_PROJECT | FK_PROJECT_USERS_PROJECT | 764     | const                    |    1 | Using index condition |
|  1 | SIMPLE      | x2    | eq_ref | PRIMARY                          | PRIMARY                  | 764     | rawls.x3.USER_SUBJECT_ID |    1 | NULL                  |
+----+-------------+-------+--------+----------------------------------+--------------------------+---------+--------------------------+------+-----------------------+
```

File: src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala [hunk 1]
```diff
@@ -14,6 +15,8 @@ case class RawlsGroupName(value: String) extends UserAuthType
 case class RawlsGroupEmail(value: String) extends UserAuthType
 case class RawlsBillingAccountName(value: String) extends UserAuthType
 case class RawlsBillingProjectName(value: String) extends UserAuthType
+case class RawlsBillingProjectMembership(projectName: RawlsBillingProjectName, role: ProjectRoles.ProjectRole)
```
Comment (jmthibault79) [inline src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala:18]: I don't like this name but I'm not coming up with anything better.

File: src/main/resources/swagger/rawls.yaml [hunk 2]
```diff
@@ -4471,3 +4504,27 @@ definitions:
       billingAccount:
         type: string
         description: the id of the billing account to use, must start with 'billingAccounts/'
+  RawlsBillingProjectMembership:
+    description: ''
```
Comment (jmthibault79) [inline src/main/resources/swagger/rawls.yaml]: - descriptions

**Comment by jmthibault79** [comment]
Looks good with a few small changes.  Do you have a link to the corresponding change on the UI side?  Orchestration should pass it through.

**Comment by dvoet** [comment]
@jmthibault79 https://github.com/broadinstitute/firecloud-orchestration/pull/239

**Pull request merged**